ccd-definition/ComplexTypes/HelpWithFeesDetails-CUI-R2-nonprod.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "ID": "HelpWithFeesDetails",
```

Comment:
Can you confirm that is the Remission Amount is showing or not?
Because I think that your screen not showing Remission Amount but according to the code, it will be showing

Reply:
It is not showing since I have not added it to CaseEventToComplexTypes